From 9e58d31423ff0fa20482f4f4d4f1ab4e85c94d08 Mon Sep 17 00:00:00 2001
From: Lars Uebernickel <lars.uebernickel@canonical.com>
Date: Tue, 12 Nov 2013 12:08:57 +0100
Subject: [PATCH] GtkMenuTracker: set action namespace correctly

gtk_menu_tracker_add_items() fetched the action-namespace from the menu
item, but didn't pass it into gtk_menu_tracker_section_new() when its
internal namespace was still NULL.

https://bugzilla.gnome.org/show_bug.cgi?id=712164
---
 gtk/gtkmenutracker.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gtk/gtkmenutracker.c b/gtk/gtkmenutracker.c
index c9e8459..6f0f4ba 100644
--- a/gtk/gtkmenutracker.c
+++ b/gtk/gtkmenutracker.c
@@ -290,7 +290,7 @@ gtk_menu_tracker_add_items (GtkMenuTracker         *tracker,
               g_free (namespace);
             }
           else
-            subsection = gtk_menu_tracker_section_new (tracker, submenu, FALSE, offset, section->action_namespace);
+            subsection = gtk_menu_tracker_section_new (tracker, submenu, FALSE, offset, action_namespace);
 
           *change_point = g_slist_prepend (*change_point, subsection);
           g_free (action_namespace);
-- 
1.8.4.3

